docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/manage-role-based-access-control-rbac/global-permissions.md

```diff
@@ -249,3 +286,82 @@ To refresh group memberships,
+
+| Category | Action | Global Admin | Standard User | Restricted Admin | Notes for Restricted Admin role |
+| -------- | ------ | ------------ | ------------- | ---------------- | ------------------------------- |
+| Local Cluster functions | Manage Local Cluster (List, Edit, Import Host) | Yes | No | No | |
+| | Create Projects/namespaces | Yes | No | No | |
+| | Add cluster/project members | Yes | No | No | |
+| | Global DNS | Yes | No | No | |
+| | Access to management cluster for CRDs and CRs | Yes | No | Yes | |
+| | Save as RKE Template | Yes | No | No | |
+| Security | | | | | |
+| Enable auth | Configure Authentication | Yes | No | Yes | |
+| Roles	| Create/Assign GlobalRoles | Yes | No (Can list) | Yes | Auth webhook allows creating globalrole for perms already present |
+| | Create/Assign ClusterRoles | Yes | No (Can list) | Yes | Not in local cluster |
+| | Create/Assign ProjectRoles | Yes | No (Can list) | Yes | Not in local cluster |
+| Users	| Add User/Edit/Delete/Deactivate User | Yes | No | Yes | |
+| Groups | Assign Global role to groups | Yes | No | Yes | As allowed by the webhook |
+| | Refresh Groups | Yes | No | Yes | |
+| PSP's | Manage PSP templates | Yes | No (Can list) | Yes | Same privileges as Global Admin for PSPs |
+| Tools | | | | | |
+| | Manage RKE Templates | Yes | No | Yes | |
+| | Manage Global Catalogs | Yes | No | Yes | Cannot edit/delete built-in system catalog. Can manage Helm library |
+| | Cluster Drivers | Yes | No | Yes | |
+| | Node Drivers | Yes | No | Yes | |
+| | GlobalDNS Providers | Yes | Yes (Self) | Yes | |
+| | GlobalDNS Entries | Yes | Yes (Self) | Yes | |
+| Settings | | | | | |
+| | Manage Settings | Yes | No (Can list) | No (Can list) | |
+| User | | | | | |
+| | Manage API Keys | Yes (Manage all) | Yes (Manage self) | Yes (Manage self) | |
+| | Manage Node Templates | Yes | Yes (Manage self) | Yes (Manage self) | Can only manage their own node templates and not those created by other users |
+| | Manage Cloud Credentials | Yes | Yes (Manage self) | Yes (Manage self) | Can only manage their own cloud credentials and not those created by other users |
+| Downstream Cluster | Create Cluster | Yes | Yes | Yes | |
+| | Edit Cluster | Yes | Yes | Yes | |
+| | Rotate Certificates	| Yes | | Yes | |
+| | Snapshot Now | Yes | | Yes | |
+| | Restore Snapshot | Yes | | Yes | |
+| | Save as RKE Template | Yes | No | Yes | |
+| | Run CIS Scan | Yes | Yes | Yes | |
+| | Add Members	| Yes | Yes | Yes | |
+| | Create Projects | Yes | Yes | Yes | |
+| Feature Charts since v2.5 | | | | | |
+| | Install Fleet | Yes | | Yes | Should not be able to run Fleet in local cluster |
+| | Deploy EKS cluster | Yes | Yes | Yes | |
+| | Deploy GKE cluster | Yes | Yes | Yes | |
+| | Deploy AKS cluster | Yes | Yes | Yes | |
+
+
+### Changing Global Administrators to Restricted Admins
```

Comment:
```suggestion
### Changing Global Administrators to Restricted Admins
```

This heading seems misleading -- we're advising the reader to change restricted admins to instead have custom roles with similar permissions, right?

Reply:
This section is one of several that I did not write or change. I would like to focus on the changed parts rather than re-writing the existing restricted-admin docs.